dev/consts.ts

```diff
@@ -1,16 +1,16 @@
 // Copyright Metatype OÜ, licensed under the Elastic License 2.0.
 // SPDX-License-Identifier: Elastic-2.0
 
-export const GHJK_VERSION = "0.2.0";
+export const METATYPE_VERSION = "0.4.5";
+export const PUBLISHED_VERSION = "0.4.4";
```

Comment:
When we bump to a stable version, should not the published version be bumped to the same version? Because, it usually means that the version will be published soon...

Reply:
Wouldn't that break the tests in this PR?